src/common/utility.h

```diff
@@ -226,6 +226,10 @@ namespace Utility {
     OCSYNC_EXPORT bool isConflictFile(const char *name);
     OCSYNC_EXPORT bool isConflictFile(const QString &name);
 
+    OCSYNC_EXPORT QString getConflictFilename(const QString &folderPath, const QString &relativeFilePath);
```

Comment:
I'd seek to have some symmetry in that API. We're a bit loosing this because one takes the folderPath and not the other. This is kind of leaking the implementation detail of the subsequent dir listing... Shouldn't both operate on full paths instead? (up to the caller to resolve this of course)